.github/workflows/ci.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
           npm install --legacy-peer-deps
 
       - name: Check npm security issues
-        run: npm audit fix
+        run: npm audit fix --legacy-peer-deps
```

Comment:
Is there any way we can get rid of the `--legacy-peer-deps` flags here ? 😢

Reply:
I tried.. The problem is this: `react-native@0.69.5` ([which is required by the current expo version](https://docs.expo.dev/versions/latest/)) requires `react@18.0.0` as a peer. `@react-native-picker@2.4.2` requires `react@16` or `react@17`. The newer version, `@react-native-picker@2.4.4`, works works with react 18 but expo complains:

```
Some dependencies are incompatible with the installed expo package version:
 - @react-native-picker/picker - expected version: 2.4.2 - actual version installed: 2.4.4
Your project may not work correctly until you install the correct versions of the packages.
To install the correct versions of these packages, please run: expo install [package-name ...]
```

We can just ignore this warning, it seems to work but it doesn't feel like an ideal solution. I will quickly commit a version with this changed (to see if there is another issue, unfortunately I cannot reproduce all of the CI peer dependency issues locally) and then we can decide :)

Reply:
Ehm, it seems there are a whole bunch of other issues ... that's what I was worried about :-/

Reply:
I think I just managed to get it working :) Not with all the latest versions but with more latest versions 😂